simpleviewinc/re-theme
# ZEN-179: Test Action

**Ticket**: [ZEN-179](https://jira.simpleviewtools.com/browse/ZEN-179)

## Context

* we want to setup a CI pipeline with Github actions
* as part of this pipeline, we want automated testing feedback

## Goal

* use a Github action to run tests on pull requests (and pushes), allowing the developer and any reviewers to check if all tests have passed before reviewing/approving a pull request

## Updates

* `.github/workflows/test.yml`
  * this is the test action that runs on pushes and pull requests

## Testing
> if you've already tested [this](https://github.com/simpleviewinc/keg-components/pull/26) PR for `keg-components`, you might not need to really test this. The action is identical.

* Create a new branch on my fork, branching off of my branch
* Make a change that causes the tests to fail
* Commit your change
* Create a pull request from your branch into my branch
* Wait a couple minutes for the action to run
* Verify the action ran and that the PR page displays a notice that the checks failed
* Commit a new change that fixes the failure you introduced
* Verify the action ran and that the PR page displays a notice that the checks succeeded
